files/lib/preinit/95_keymap

```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Nethesis
```

Comment:
If you want to add an header, please see https://github.com/NethServer/nextsecurity/blob/master/README.md#license-headers

Reply:
Do we need a license in a one-line file? :-)